.github/workflows/publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish Packages
```

Comment:
Although I like github-actions, we have committed to using Circle CI for the Go part of this repository already, and I would prefer to keep things consistent. Can you change this to a Circle CI routine?

Reply:
Not too familiar with CircleCI but gave it my best shot.